packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -1190,6 +1193,9 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
   @override
   String get searchWebButtonLabel => 'Search Web';
 
+  @override
+  String get shareButtonLabel => 'Share...';
```

Comment:
can't remember - does native ios has the `...`?

Reply:
Yep! 
![image](https://github.com/flutter/flutter/assets/36148254/2d8e9d55-c537-4c5e-8af5-6f20f36372ed)